datastore/google/cloud/datastore/key.py

```diff
@@ -281,6 +298,47 @@ def to_protobuf(self):
 
         return key
 
+    def to_legacy_urlsafe(self):
+        """Convert to a base64 encode urlsafe string for App Engine.
+
+        This is intended to work with the "legacy" representation of a datastore
+        "Key" used within Google App Engine (a so-called "Reference"). This is
+        intended as a drop in for the value returned when using
+        ``ndb.Key(...).urlsafe()``.
+
+        :rtype: bytes
+        :returns: ASCII bytes contain the key encoded as URL-safe base64.
```

Comment:
This reads funny to me, why not just `A bytestring containing the key encoded as URL-safe base64`